drivers/gles3/shaders/scene.glsl

```diff
@@ -725,8 +731,8 @@ void light_compute(vec3 N, vec3 L, vec3 V, float A, vec3 light_color, float atte
 #else
 		float D = D_GGX(cNdotH, alpha_ggx);
 		float G = V_GGX(cNdotL, cNdotV, alpha_ggx);
-#endif // LIGHT_ANISOTROPY_USED
-	   // F
+#endif // LIGHT_ANISOTROPY_USED \
```

Comment:
typo?

Reply:
Yeah I guess, but a weird one, I suspect it might have been my version of clang not making sense of the `// F` comment on the next line. I'm guessing that comment is bogus anyway

Reply:
Yeah turned out to be clang.. sigh